examples/scripts/color/atom-data.js

```diff
@@ -0,0 +1,29 @@
+/** Demonstration of data colormaker - passing arbitrary data through to the
+ * colormaker for plotting.
+ */
+stage.loadFile('data://1blu.mmtf').then(function (c) {
+  // We might be colouring by some calculated/predicted property (e.g. ML model which outputs
+  // scores/quantities for individual atoms)
+  // To keep this self-contained, we'll just calculate distance from the origin for each
+  // atom:
+
+  var structure = c.structure
+  var distance = new Float64Array(structure.atomCount)
+
+  structure.eachAtom(ap => {
+    distance[ap.index] = Math.sqrt(ap.x ** 2 + ap.y ** 2 + ap.z ** 2)
+  })
+
+  var repr = c.addRepresentation('ball+stick', {
+    color: 'structuredata',
+    colorData: {atomData: distance},
+    colorDomain: [30.0, 60.0]
+  })
+  c.autoView()
+
+  // If colorData isn't defined, empty etc, falls back to value, here we wait 2 seconds
+  // then update the paramter to an empty array to demonstrate
+  window.setTimeout(() => {
```

Comment:
Not sure if it's a good idea to remove a coloring when demonstrating the feature. It might give a wrong impression.

Reply:
Yeah, I agree, I was being lazy - the idea was to show that the color falls back to `colorValue` if data is missing from the input, but a better way is probably to just provide some partial data in the first instance - will change this.